packages/react-composites/src/composites/CallWithChatComposite/CallWithChatControlBar.tsx

```diff
@@ -74,6 +75,12 @@ const inferCallWithChatControlOptions = (
  * @private
  */
 export const CallWithChatControlBar = (props: CallWithChatControlBarProps & ContainerRectProps): JSX.Element => {
+  useEffect(() => {
+    // On mount, button used for initial focus is hidden to prevent screen readers from announcing the initial focus
+    if (document.querySelector('[data-ui-id=call-with-chat-autofocus-hidden-button]') === document.activeElement) {
+      document.activeElement?.toggleAttribute('hidden');
```

Comment:
why `toggleArtibute` and not set attribute? I assume there's not scenario where we toggle it back to showing?